src/Tooltip/TooltipDictionary.yaml

```
@@ -1472,22 +1472,22 @@ cvData_timeout:
         [Learn More](https://docs.harness.io/article/3xhqq9xllp-verify-deployments-with-the-verify-step#step_2_enter_a_name_and_timeout)
     width: "400"
 cvData_spec.type:
-    content: In Continuous Verification Type, select a type that matches your deployment strategy. Rolling, Canary, Blue Green, and Load Test simply refer to the strategy in the stage. [Learn More](https://docs.harness.io/article/3xhqq9xllp-verify-deployments-with-the-verify-step#step_3_select_a_continuous_verification_type)
+    content: In Continuous Verification Type, select a type that matches your deployment strategy. Rolling, Canary, Blue Green, and Load Test simply refer to the strategy in the stage. [Learn More](https://developer.harness.io/docs/continuous-delivery/verify/verify-deployments-with-the-verify-step#continuous-verification-type)
     width: "400"
 cvData_spec.monitoredServiceRef:
-    content: Harness automatically creates a Monitored Service using a concatenation of the Service and Environment names. In Monitored Service, click Click to autocreate a monitored service. If the stage Service or Environment settings are Runtime Inputs, the Monitored Service and Health Sources settings will show up in the Runtime Input settings when you run the Pipeline. [Learn More](https://docs.harness.io/article/3xhqq9xllp-verify-deployments-with-the-verify-step#step_4_create_a_monitored_service)
+    content: Harness automatically creates a Monitored Service using a concatenation of the Service and Environment names. In Monitored Service, click to autocreate a monitored service. If the stage Service or Environment settings are Runtime Inputs, the Monitored Service and Health Sources settings will show up in the Runtime Input settings when you run the Pipeline.
```

```suggestion
    content: Harness automatically creates a Monitored Service using a concatenation of the Service and Environment names. In Monitored Service, click to autocreate a monitored service. If the stage's Service or Environment settings are runtime inputs, then the Monitored Service and Health Sources settings appear in the runtime input settings when you run the pipeline.
```

This is unclear: "In Monitored Service, click to autocreate a monitored service" What do you click?